doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -252,6 +252,8 @@ nRF Desktop
     * :kconfig:option:`CONFIG_FPROTECT`: used to protect the bootloader partition against memory corruption
     * :kconfig:option:`CONFIG_HW_STACK_PROTECTION`: used to protect against stack overflows
 
+  * MCUboot bootloader configuration for the MCUboot SMP build type and the nRF52840 Gaming Mouse target to enable the :kconfig:option:`CONFIG_ARM_MPU` Kconfig option that is used to enable Memory Protection Unit (MPU).
```

Comment:
Maybe instead `that is used to enable Memory Protection Unit (MPU)` we could state `that is needed to enable hardware stack protection`? (fact that it enables ARM MPU is obvious looking at Kconfig name :) )